src/k8s/pkg/k8sd/types/cluster_config_features.go

```diff
@@ -35,7 +35,7 @@ type LocalStorage struct {
 	Enabled       *bool   `json:"enabled,omitempty"`
 	LocalPath     *string `json:"local-path,omitempty"`
 	ReclaimPolicy *string `json:"reclaim-policy,omitempty"`
-	SetDefault    *bool   `json:"set-default,omitempty"`
+	Default       *bool   `json:"default,omitempty"`
 }
```

Comment:
Of note, this `json:"set-default"` -> `json:"default"` is also a breaking change, which would make the value `nil` in existing clusters.

In the future, we could keep the `json:"set-default"` around, and just rename the struct field.

But again, seeing as there are no stable releases yet, we're good to go with the change.